provider/cpu_utils.py

```diff
@@ -99,3 +102,32 @@ def check_guest_cpu_topology(session, os_type, cpuinfo):
         logging.debug("CPU infomation of guest:\n%s", out)
 
     return is_matched
+
+
+def check_flags(params, flags, test, session=None):
+    """
+    Check cpu flags on host or guest.(only for Linux now)
+    :param params: Dictionary with the test parameters
+    :param flags: checked flags
+    :param test: QEMU test object
+    :param session: guest session
+    """
+    cmd = params["check_flag_cmd"]
+    func = process.getoutput
+    if session:
+        func = session.cmd_output
+    out = func(cmd).split()
+    missing = [f for f in flags.split() if f not in out]
+    if session:
+        error_context.context("Check cpu flags inside guest", logging.info)
```

Comment:
Directly using `logging.info()` if not `@error_context` for the function.

Reply:
> Directly using `logging.info()` if not `@error_context` for the function.

update and thanks.